spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
```

Comment:
This is the spec that caught #13632. And also it makes sure this relationship is maintained when the code base is evolved.